lib/render/site-award.php

```diff
@@ -128,6 +136,20 @@ function RenderAwardGroup($awards, $title): void
     echo "</div>";
 }
 
+function RenderCounter($icon, $text, $numItems, $numHidden): string
+{
+    $tooltip = $icon . "  " . $numItems . " " . $text;
+    if ($numHidden > 0) {
+        $tooltip .= "【" . $numHidden . " hidden】";
```

Comment:
Recommend unstylized parentheses.

Reply:
I'm changing it to [square brackets] then, if that's okay with you.

Reply:
I'm for normal paranthesis ;)

Reply:
So 2-1 for normal parenthesis then 🗳️